db/src/models/events.rs

```diff
@@ -1101,7 +1101,11 @@ impl Event {
         let ticket_id: Option<Uuid> = None;
         let tickets = diesel::sql_query(q)
             .bind::<Nullable<dUuid>, _>(Some(self.id))
-            .bind::<Nullable<Text>, _>(query.map(|q| str::replace(&q, " ", "%")))
+            .bind::<Nullable<Text>, _>(
+                query
+                    .map(|q| str::replace(&q, " ", "%"))
```

Comment:
I'm not sure how much of a performance knock we would take, but what do you think about doing a:
```suggestion
                    .map(|q| str::replace(&q, " ", "").split("").collect::<Vec<&str>>().join("%"))
```

Reply:
I doubt it'd use an index regardless since we're wrapping the query in wildcards anyhow so probably not a huge deal performance wise regardless. That's to allow no spaces between the first and last name?

Reply:
Didn't break the tests and seems fine so just switched to that approach. Gives us more flexibility with the search at least.